crates/rethnet_eth/src/remote.rs

```diff
@@ -393,24 +393,23 @@ impl RpcClient {
 
 #[cfg(test)]
 mod tests {
-    use super::*;
-
     use std::str::FromStr;
 
     use crate::{Address, Bytes, U256};
 
-    fn get_alchemy_url() -> Result<String, String> {
-        Ok(std::env::var_os("ALCHEMY_URL")
+    use super::*;
+
+    fn get_alchemy_url() -> String {
+        std::env::var_os("ALCHEMY_URL")
             .expect("ALCHEMY_URL environment variable not defined")
             .into_string()
-            .expect("couldn't convert OsString into a String"))
+            .expect("Couldn't convert OsString into a String")
     }
 
+    #[test_with::env(ALCHEMY_URL)]
```

Comment:
@feuGeneA I added a macro that ignores tests when the `ALCHEMY_URL` environment variable is not set. This allows people have local tests succeed even if they don't have alchemy setup